commit 3c0d0894320cc517fda657c69939cd0313d0b4e2
Author: Ilya Dryomov <idryomov@gmail.com>
Date:   Mon Jun 21 11:53:38 2021 +0200

    libceph: don't pass result into ac->ops->handle_reply()
    
    There is no result to pass in msgr2 case because authentication
    failures are reported through auth_bad_method frame and in MAuth
    case an error is returned immediately.
    
    Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
    Reviewed-by: Jeff Layton <jlayton@kernel.org>

diff --git a/net/ceph/auth_none.c b/net/ceph/auth_none.c
index 70e86e462250..aab490a111eb 100644
--- a/net/ceph/auth_none.c
+++ b/net/ceph/auth_none.c
@@ -69,7 +69,7 @@ static int build_request(struct ceph_auth_client *ac, void *buf, void *end)
  * the generic auth code decode the global_id, and we carry no actual
  * authenticate state, so nothing happens here.
  */
-static int handle_reply(struct ceph_auth_client *ac, int result,
+static int handle_reply(struct ceph_auth_client *ac,
 			void *buf, void *end, u8 *session_key,
 			int *session_key_len, u8 *con_secret,
 			int *con_secret_len)
@@ -77,7 +77,7 @@ static int handle_reply(struct ceph_auth_client *ac, int result,
 	struct ceph_auth_none_info *xi = ac->private;
 
 	xi->starting = false;
-	return result;
+	return 0;
 }
 
 static void ceph_auth_none_destroy_authorizer(struct ceph_authorizer *a)